src/styles/header.scss

```diff
@@ -0,0 +1,63 @@
+@import 'variables';
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 60px;
+  padding: 0 50px;
+  background: #fff;
+
+  &__logo {
+    height: 40px;
+    width: 40px;
+  }
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+  }
+
+  &__item {
+    margin-left: 20px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    font-size: 12px;
+    font-weight: 500;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $main-accent-color;
```

Comment:
The `transition-duration` should be set to `$transition-duration` instead of `transition-duration` to correctly reference the SCSS variable. Ensure that the variable `$transition-duration` is defined in your `variables.scss` file.